tests/lapack_like/TSVD.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+   Copyright (c) 2009-2015, Jack Poulson
+   All rights reserved.
+
+   This file is part of Elemental and is under the BSD 2-Clause License, 
+   which can be found in the LICENSE file in the root directory, or at 
+   http://opensource.org/licenses/BSD-2-Clause
+*/
+#include "El.hpp"
+using namespace El;
+
+template< typename Matrix>
+class AOp {
+
+    public:
+    AOp( const Matrix& A_, Orientation o_ = El::NORMAL) : A( A_), o(o_) {}
+    
+    template< typename V>
+    void operator()( V& v) const {
+        V w;
+        Gemv(o, 1.0, A, v, w);
+        v = w;
+    }
+
+    Int Height() const {
+        if( o == El::NORMAL){ return A.Height(); }
+        return A.Width();
+    }
+
+    Int Width() const {
+        if( o == El::NORMAL){ return A.Width(); }
+        return A.Height();
+    }
+
+    const Matrix& A;
+    const Orientation o = El::NORMAL;
+}; //Matrix Operator
+
+
+template<typename F>
+void TestTSVD( const DistMatrix<F>& A, Int k=3){
+    typedef Base<F> Real;
+    const auto& g = A.Grid();
+    int m = A.Height();
+    int n = A.Width();
+    DistMatrix<F> Uk( g), Sk( g), Vk( g);
+    DistMatrix<F> U( g), V( g);
+    DistMatrix<Base<F>, VR, STAR> S( g);
+    if( g.Rank() == 0 )
+        Output("  Starting TSVD factorization...");
+    const double startTime = mpi::Time();
+    typedef AOp<DistMatrix<F>> Operator; 
+    Operator Aop( A);
+    Operator AAdjop( A, El::ADJOINT);
+    TSVD( m, n, Aop, AAdjop, k, Uk, Sk, Vk);
+    const double runTime = mpi::Time() - startTime;
+    Output("TSVD Time = ",runTime);
+    mpi::Barrier( g.Comm() );
+    if( g.Rank() == 0 )
+        Output("  Starting SVD factorization..."); 
+    SVD( A, U, S, V);  
+    mpi::Barrier( g.Comm() );
+    for( Int i = 0; i < k; ++i){
+        std::cout <<  i << ": ";
+        const Real& A = Sk.Get(i,0);
+        const Real& B = S.Get(i,0);
+        Output( "Sk = ", A, "SVD: S_k", B);
+	auto eigenvalue_error = A - B;
+        auto left_svec_error = Norm(Abs(Uk( ALL, IR(i))) - Abs(U( ALL, IR(i)))); 
+        auto right_svec_error = Norm(Abs(Vk( ALL, IR(i))) - Abs(V( ALL, IR(i)))); 
+        if( eigenvalue_error > limits<Real>::Epsilon()){
```

Comment:
I think you mean `El::limits::Epsilon<Real>()`; `std::numeric_limits<Real>` does not support many of Elemental's datatypes (e.g., `BigFloat`). The syntax is slightly different and this is the first problem breaking your Travis build.

It would also be a good idea to use a more principled failure condition, such as `max(m,n) || A ||_F eps`